fastlane/lanes/localization.rb

```diff
@@ -156,13 +156,13 @@
     files = {
       release_note: File.join(metadata_folder, 'release_notes.txt'),
       release_note_short: File.join(metadata_folder, 'release_notes_short.txt'),
-      'short-description': File.join(metadata_folder, 'short_description.txt'),
-      'app-store-description': File.join(metadata_folder, 'full_description.txt'),
-      'app-store-name': File.join(metadata_folder, 'title.txt'),
+      play_store_promo: File.join(metadata_folder, 'short_description.txt'),
+      play_store_desc: File.join(metadata_folder, 'full_description.txt'),
+      play_store_app_title: File.join(metadata_folder, 'title.txt'),
```

Comment:
Suggestion (💡): Consider ordering those 3, `app_title`, `promo` and `desc` in that order, just like it is done elsewhere. And actually, even if at some places it is not ordered as such (ie. line `125`), consider reordering those too, just so that they end up being consistent.
- `play_store_app_title`
- `play_store_promo`
- `play_store_desc`

Reply:
Good idea! Addressed in 3c1251d66034b311d7b045330ab95ff9b32bb2dc

Reply:
Awesome, thanks for updating the `localization.rb` as well! 🤟

Reply:
To be honest that odd order in the `.po` file was bothering me for some time as well 😛 So it was high time to make it more consistent 😄